src/main/java/tools/map/making/MapCreator.java

```diff
@@ -379,14 +352,11 @@ public void run() {
       if (runUtilitiesAsSeperateProcesses) {
         runUtility(TileImageBreaker.class);
       } else {
-        (new Thread() {
-          @Override
-          public void run() {
-            try {
-              TileImageBreaker.main(new String[0]);
-            } catch (final Exception e) {
-              ClientLogger.logQuietly(e);
-            }
+        new Thread(() -> {
+          try {
+            TileImageBreaker.main(new String[0]);
+          } catch (final Exception e1) {
+            ClientLogger.logQuietly(e1);
```

Comment:
When I first saw these `e1` ... `e5` variable names, I thought that by introducing the lambdas, somehow all this code was being pulled up into the same scope, and the renames were an attempt to avoid duplicate local variables.  Upon further inspection, it seems it's just to avoid a collision with each lambda's `Event` parameter, which is already named `e`.

it might be less confusing to rename all the exception variables `ex` instead of using the sequence `e1` ... `e5`.  In fact, you'll see someone already did that on line 457 below when they introduced `TileImageReconstructor`.

Reply:
Good catch, changed that!